tests/MSBuildDeviceIntegration/Tests/DeploymentTest.cs

```diff
@@ -62,12 +62,21 @@ public void CheckXamarinFormsAppDeploysAndAButtonWorks ()
 			if (!HasDevices)
 				Assert.Ignore ("Skipping Test. No devices available.");
 			AdbStartActivity ($"{proj.PackageName}/md52d9cf6333b8e95e8683a477bc589eda5.MainActivity");
-			WaitForActivityToStart (proj.PackageName, "MainActivity", output: out string output, timeout: 10);
+			string activityStartLogcatPath = Path.Combine (XABuildPaths.TestOutputDirectory, builder.ProjectDirectory, "startup-logcat.log");
```

Comment:
These tests we usually use the `Root` property: https://github.com/xamarin/xamarin-android/blob/ecc016a2157e2a048dfd08f57e5ce4952aae3127/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/BaseTest.cs#L133-L137

```suggestion
			string activityStartLogcatPath = Path.Combine (Root, builder.ProjectDirectory, "startup-logcat.log");
```